src/test/java/com/mariamkatamashvlii/gym/daoImplTests/TraineeDaoImplTest.java

```diff
@@ -0,0 +1,92 @@
+package com.mariamkatamashvlii.gym.daoImplTests;
+
+import com.mariamkatamashvlii.gym.daoImpl.TraineeDaoImpl;
+import com.mariamkatamashvlii.gym.model.Trainee;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TraineeDaoImplTest {
+    private TraineeDaoImpl traineeDaoImpl;
+    private Map<Long, Trainee> traineeMap;
+    private Trainee trainee;
```

Comment:
Incorrect approach for Spring.

You can uset @Mock and @InjectMocks annotations.
It will mock your beans